src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -152,7 +152,7 @@ def login(self,
               allow_no_subscriptions=False,
               use_cert_sn_issuer=None,
               show_progress=False,
-              **kwargs):
```

Comment:
This PR removes lots of `kwargs`. I just want to confirm that `kwargs` is never used and always empty, right?